src/app/components/BCP47Tag.svelte

```diff
@@ -1,11 +1,18 @@
 <script lang="ts">
   import bcp47Parser from "bcp-47";
+  import worker from "../spawn-worker";
 
   const TAG_INPUT_ID = "tag-input";
 
   let bcp47Tag = "";
+  let validTag = false;
   $: schema = bcp47Parser.parse(bcp47Tag);
-  $: invalidTag = bcp47Tag !== "" && schema.language === null;
+  $: validTag = bcp47Tag !== "" && schema.language === null;
```

Comment:
The BCP-47 tag is valid when it is has content (not empty) and it has a parsed `language`:

```suggestion
  $: validTag = bcp47Tag.length > 0 && schema.language != null;
```